server/src/categories/categories.controller.ts

```diff
@@ -0,0 +1,95 @@
+import {
+    Controller,
+    Get,
+    Post,
+    Body,
+    Patch,
+    Param,
+    Delete,
+    UseGuards,
+    HttpException,
+    HttpStatus,
+  } from '@nestjs/common';
+  
+  import { CategoriesService } from './categories.service';
+  import { CreateCategoryDto } from './dto/create-category';
+  import { UpdateCategoryDto } from './dto/update-category';
+  // import { JwtAuthGuard } from '../auth/guards/jwt-auth.guard';
+  //authGaurd excluded because it is outside of the scope of this PR

```

Comment:
comments go above the line they refer to
better comment would be
`// TODO ticket for adding auth guards https://github.com/Nonprofit-Exchange-Hub/web-app/issues/84`